assets/dev/scss/frontend/_container.scss

```diff
@@ -164,15 +163,13 @@
 		}
 	}
 
-	.elementor & {
-
-		&,
-		& > .e-container__inner {
-
-			// This was done to make sure that the widgets inside containers do not exceed the container's max width.
-			& > .elementor-widget {
-				max-width: 100%;
-			}
+	// Hack with repeated classnames to increase the specificity.
+	.elementor.elementor &,
```

Comment:
@udidol 

The `custom width` control for the widgets sets the `width` and `max-width` value. 

Inside the container the `max-width` value is causing problems because of two aspects: 
1. The value can be larger than 100% of the screenwidth, which makes the horizontal scrollbar to appear.
2. The `max-width` prevents the user from user the `flex-grow` control.

I have increased the specificity to overwrite that `max-width` value created by the widget `max-width` settings.